src/net/sourceforge/kolmafia/maximizer/Evaluator.java

```diff
@@ -249,27 +243,27 @@ private void parse(String expr) {
       if (keyword.startsWith("\"") && keyword.endsWith("\"")) {
         keyword = keyword.substring(1, keyword.length() - 1).trim();
       }
-      if (keyword.equals("min")) {
-        if (index != null) {
-          this.min.put(index, weight);
-        } else {
-          this.totalMin = weight;
+      switch (keyword) {
```

Comment:
very iffy on this being an improvement. It's in the middle of a list of "starts with / equals" checks, so you need multiple switches... I think using switches instead of equals makes it slightly harder to scan.

Reply:
Might be debatable but can be considered next time someone cleans up Evaluator